docs/en-US/adding-a-new-site/custom-nginx.md

```diff
@@ -10,4 +10,25 @@ By default, VVV copies a `vvv-nginx.conf` file into place. The site templates cr
 
 This is useful if you wish to implement or change Nginx rules. For example, using a subdirectory installation of WordPress.
 
-It may also be necessary to replace the nginx wp common rules. VVV does this using an include statement, but you could replace this with the contents of that file. It can be found in `config/nginx-config/nginx-wp-common.conf`. Note that VVV will only copy the final Nginx config file, it won't copy over includes.
\ No newline at end of file
+It may also be necessary to replace the nginx wp common rules. VVV does this using an include statement, but you could replace this with the contents of that file. It can be found in `config/nginx-config/nginx-wp-common.conf`. Note that VVV will only copy the final Nginx config file, it won't copy over includes.
+
+## About the Nginx generation
```

Comment:
I think this sections should be named something like:

> ## VVV Nginx Substitutions

And take a more reference like form, with a list of things that get substituted, I don't know that this style works so well in written english, maybe spoken

Reply:
Added a new line that explain the purpose of the section